happy-travel/edo
# Feature/aa 1872 child agencies pays by root balance

Issue: https://github.com/happy-travel/agent-app-project/issues/1872

Test cases: 
 1) Check if Agent and Admin endpoints for getting agencies' accounts retrieve root's accounts for childs agencies
 2) Check if book step for child agencies retrieves root accounts
 3) Check if charge step within deadline while "book by account" retrieves root accounts
 4) Check if charge step before deadline from kanazawa retrieves root accounts
